spec/models/item_spec.rb

```diff
@@ -5,8 +5,19 @@
                 item.categories << category
                 item.save
                 item}
+  let(:item2) { item = Fabricate.build(:item)
+               item.categories << category
+               item.save
+               item}
```

Comment:
Expression at 11, 20 should be on its own line.<br>Inconsistent indentation detected.